lib/active_model/nested_attributes.rb

```diff
@@ -22,6 +23,11 @@ def accepts_nested_attributes_for(*models, **_options)
 
       private
 
+      def validate_options!(options)
+        opts = { allow_destroy: true }.merge(options)
+        opts.assert_valid_keys(:allow_destroy)
+      end
```

Comment:
Allow developers to toss the `allow_destroy` arg at nested attributes and no-op. Raise an exception if any other options passed.

In practice, the nested attrs implementation in H3 allows all nested attributes to be destroyed.